conbench/app/benchmarks.py

```diff
@@ -236,6 +246,9 @@ def _get_benchmark_and_run(self, benchmark_id):
 
 class BenchmarkList(AppEndpoint, ContextMixin):
     def page(self, benchmarks):
+        # Note(JP): What type is benchmarks? As of `response =
+        # self.api_get("api.benchmarks")` down below it's seemingly the result
+        # of JSON-decoding, i.e. not the DB model type.
```

Comment:
yeah from what I've seen, most python modules in `conbench/app/` only work with dicts (the responses from the API), not the entities